app/experimenter/docs/adrs/0008-integrating-htmx.md

```diff
@@ -0,0 +1,73 @@
+# Front-end web development with HTMX
+
+- Status: accepted
+- Deciders: Yashika Khurana, Jared Lockhart, Elise Richards
+- Date: 2022-10-25
+
+## Context and Problem Statement
+
+Nimbus UI uses GraphQL to fetch data from the backend and it highly depends on the two packages i.e. [graphene-django](https://github.com/graphql-python/graphene-django) and [graphene](https://github.com/graphql-python/graphene). We have found out that using GraphQL significantly hinders the performance of the Experimenter as it adds extra overhead while making the call and the benefit of using GraphQL is not something we are making use of. As our application is growing so what steps we should take to improve the performance and better maintainability of the application?
+
+## Decision Drivers
```

Comment:
Is one of the drivers also to improve the maintainability of the codebase?

Reply:
yes! Seeing the future of the Experimenter